docs/marketplace/guides/publish-offer.md

```
@@ -3,4 +3,149 @@ id: "publish-offer"
+<br/>
+
+2.2. Complete all the fields in the appearing window.
+
+<img src={require('../images/new-offer.png').default} width="auto" height="auto" border="1"/>
+<br/>
+<br/>
+
+- **Name**: Type in the model name. Providing a meaningful name makes it easier to find the offer on the Marketplace.
+- **Category**: Select the type of task that the model is designed to perform.
+- **Engine**: Select compatible engines; it is recommended to select both variants:
+    + **GPU-only** engines require CPU+GPU compute offers.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: plus

(MD004, ul-style)